<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowTrackSectorEditor</class>
 <widget class="QDialog" name="WindowTrackSectorEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Track Sectors</string>
  </property>
  <widget class="QGroupBox" name="gb_TrackSelection">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Selected track and configuration:</string>
   </property>
   <widget class="QLineEdit" name="le_TrackName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>None</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_TrackSelect">
    <property name="geometry">
     <rect>
      <x>299</x>
      <y>30</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
  </widget>
  <widget class="QListWidget" name="lb_MainList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>371</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Remove">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_SelectTs">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>290</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_LbHeading">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Track Sectors:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gb_AddNew">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>371</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Add new track sector:</string>
   </property>
   <widget class="QLabel" name="lbl_Name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Start">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Start:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_End">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>End:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_Name">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_Start">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_End">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Add">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
